src/main/java/com/autotune/common/k8sObjects/TrialSettings.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Red Hat, IBM Corporation and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package com.autotune.common.k8sObjects;
+
+import com.google.gson.annotations.SerializedName;
+
+public class TrialSettings {
```

Comment:
Trial settings needs to have all of the following fields as per this JSON
https://github.com/kruize/autotune/blob/mvp_demo/examples/em-json/GeneralPerfExperimentsTemplate.json
```
       "trial_settings": {
          "measurement_cycles": "3",
          "warmup_duration": "1min",
          "warmup_cycles": "3",
          "measurement_duration": "1min",
          "iterations": "3"
```